module2/homework/addEven/addEven.hpp

```diff
@@ -1,8 +1,11 @@
-#pragma once
 #include <vector>
 
 int addEven(const std::vector<int>& numbers) {
-    // TODO: Your implementation goes here
-    // Below return is only to make this function compile now
-    return -1;
+
+    int result = 0;
+
+    for(const int& element : numbers) {
+        result += element % 2 == 0 ? element : 0;
```

Comment:
That's much more obscure than a simple if statement.

Reply:
maybe, but it's working :)

Reply:
"But it works" is the last thing i want in a code review ;)

Reply:
For me this is correct. operator `: ?` is good option here to simplify.

Reply:
For me it looks unclear :) At least some parenthesis could make this code look better.
```suggestion
        result += (element % 2 == 0) ? element : 0;
```